Classes/CardIOCardScanner.mm

```diff
@@ -76,7 +93,32 @@ - (void)addFrame:(CardIOIplImage *)y
   result.flipped = flipped;
   scanner_add_frame_with_expiry(&_scannerState, y.image, scanExpiry, &result);
   self.lastFrameWasUsable = result.usable;
-  if(!result.usable) {
+  
+  if(!result.usable)
+  {
```

Comment:
The style used within this codebase is to keep opening braces on the previous line. Please stick to this style. (Doesn't apply, of course, when you bring whole new libraries into the project.)